src/cli/type-generator.js

```diff
@@ -85,6 +88,9 @@ module.exports = class TypeGenerator {
       return abis.map((abi, name) => this._generateTypesForABI(abi))
     } catch (e) {
       this.logger.fatal('Failed to generate types for contract ABIS:', e)
+    } finally {
+      this.logger.info('')
+      this.logger.info(chalk.green('Types generated'))
```

Comment:
This message breaks with the convention of using green message only for the top-level steps. Let's not include it at all.

Reply:
Ok. The reason I added it was because there was no confirmation of type generation completing when not in verbose mode.  Ok to have no logging in that situation? Or perhaps another way to include it, maybe as a different color?